src/AppInstallerCLICore/Workflows/ShowFlow.cpp

```diff
@@ -12,17 +12,57 @@ using namespace AppInstaller::Utility;
 using namespace AppInstaller::Utility::literals;
 
 namespace {
-    void ShowMultiLineField(Execution::OutputStream& outputStream, AppInstaller::StringResource::StringId label, std::string& value)
+
+    template <typename String>
+    void ShowSingleLineField(Execution::OutputStream outputStream, const AppInstaller::StringResource::StringId& label, const String& value, bool indent = false)
+    {
+        if (value.empty())
+        {
+            return;
+        }
+        if (indent)
+        {
+            outputStream << "  "_liv;
+        }
+        outputStream << Execution::ManifestInfoEmphasis << label << ' ' << value << std::endl;
+    }
+
+    template <typename String>
+    void ShowMultiLineField(Execution::OutputStream outputStream, const AppInstaller::StringResource::StringId& label, const String& value)
     {
-        bool isMultiLine = FindAndReplace(value, "\n", "\n  ");
+        if (value.empty())
+        {
+            return;
+        }
+        /*
+            We need to be able to find and replace within the string.However, we don't want to own the original string
+            Therefore, a copy is created here so we can manipulate it. The memory should be freed again once this method
+            returns and the string is no longer in scope.
+        */
+        std::string m_value = value;

```

Comment:
nit: m_value is not a class member, maybe name it shownValue or something better?